src/intervals/interval_operations/bisect.jl

```diff
@@ -42,3 +56,29 @@ function mince(x::Interval{T}, n::Integer) where {T<:NumTypes}
     end
     return v
 end
+
+mince(x::AbstractVector, n::Integer) = mince(x, ntuple(_ -> n, length(x)))
+
+mince(x::AbstractVector, n::NTuple{N,Integer}) where {N} = mince!(Vector{typeof(x)}(undef, prod(n)), x, n)
+
+"""
+    mince!(v, x, n)
+
+In-place version of [`mince`](@ref).
+"""
+mince!(v::AbstractVector, x::AbstractVector, n::Integer) = mince!(v, x, ntuple(_ -> n, length(x)))
+
+function mince!(v::AbstractVector, x::AbstractVector, n::NTuple{N,Integer}) where {N}
```

Comment:
Probably not, it will fail when calling `resize!` probably. Since it is an in-place functionality, one should not use non-mutable array types; this type of signature occurs also in LinearAlgebra.